### PR TITLE
Added support for the magic link adapter

### PR DESCRIPTION
**Pull request by mketchum**

### CODE REVIEW

File: docs/supportedIntegrations.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ The widget is regularly updated to support more integrations.
 | [Identifier First](https://docs.pingidentity.com/csh?Product=pf-latest&topicname=iek1564003022460.html)                                                    | :heavy_check_mark:    | :heavy_check_mark:  |
 | [Jamf Integration Kit](https://docs.pingidentity.com/bundle/integrations/page/wyg1612987868528.html)                                                       | :heavy_check_mark:    | :heavy_check_mark:  |
 | [LinkedIn Cloud Identity Connector](https://docs.pingidentity.com/bundle/integrations/page/pwm1563995028175.html)                                          | :heavy_check_mark:    | :heavy_check_mark:  |
+| [Magic Link Integration Kit](https://docs.pingidentity.com/)                      | :heavy_check_mark:    | :heavy_check_mark:  |
```
Comment (mketchum) [inline docs/supportedIntegrations.md:26]: Will fix when the link is ready

File: src/index.js [hunk 2]
```diff
@@ -1223,6 +1234,35 @@ export default class AuthnWidget {
...
+    this.store.dispatch('POST_FLOW', 'continue', '{}')
+      .catch(() => this.generalErrorRenderer(AuthnWidget.COMMUNICATION_ERROR_MSG));
+  }
+
+  pollLinkStatus() {
+    this.pollLink(5000);
+  }
+
+  async pollLink(timeout) {
+    let fetchUtil = this.store.fetchUtil;
```
Comment (jkhosravian-ping) [inline src/index.js:1247]: I'll need to change this method in the near future :) can you introduce a new method in the store to do the bulk of this operation? aka perform a poll and return the state.

**Review by jwong-ping — APPROVED** [review]
I think the polling frequency is a bit long, but I see that's how it is elsewhere.

**Review by jkhosravian-ping — APPROVED** [review]

**Review by anandgopinath-ping — DISMISSED** [review]

File: src/index.js (hunk 2 above)
Comment (mketchum) [reply src/index.js]: Added a new poll() method to the datastore for this functionality. Let me know if that's not what we are looking for.

Comment (jkhosravian-ping) [reply src/index.js]: Yea, thanks so much.

**Review by jkhosravian-ping — APPROVED** [review]

**Review by jwong-ping — APPROVED** [review]

**Pull request merged**